src/main/kotlin/com/example/ajaxproject/service/interfaces/UserService.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.ajaxproject.service.interfaces
+
+import com.example.ajaxproject.dto.UserDTO
+import com.example.ajaxproject.model.User
+
+interface UserService {
+
+    fun createUser(userDTO: UserDTO): User
+
+    fun updateUser(id: Long, userDTO: UserDTO): User
+
+    fun deleteUser(id: Long)
+
+    fun findUserById(id: Long): User?
+
+    fun findAllUsers(): List<User>
+
+}
```

Comment:
no new line at the end of file
![image](https://github.com/OleksandRomaniuk/Ajax/assets/116563306/d16b768b-98f0-4e9b-a134-ed7eccb29525)


Please setup your IDE to add line break at the end of file automatically